config/services.php

```diff
@@ -64,6 +64,12 @@
 
         // https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html
         EnvironmentEntity::TYPE_MAGENTO2 => [
+            '2.4.6' => [
+                'database' => ['mariadb:10.6', 'mysql:8.0-oracle'],
+                'elasticsearch' => ['ajardin/magento2-elasticsearch:7.17'],
```

Comment:
Shouldn't be the 8.4 version instead?
```suggestion
                'elasticsearch' => ['ajardin/magento2-elasticsearch:8.4'],
```

Reply:
Both https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/system-requirements.html
Let you squash commits during the merge action